src/domain/utils.ts

```diff
@@ -0,0 +1,7 @@
+import { RepositoryError, UnknownRepositoryError } from "./errors"
+
+export const isRepoError = (obj): boolean =>
```

Comment:
I have thought about it and don't think we should generally not be checking for specific errors inside the `domain/core` code.
Just:
```
const thing = serviceCall()
if (thing instances Error) {
  return thing
}
```

We should only check for a `SpecificError` when the type is actually relevant. For example when translating the errors back to the user in the GQL layer or something like that.

Reply:
done